src/marionette.sliding-view.js

```diff
@@ -24,6 +24,9 @@ Mn.SlidingView = Mn.CollectionView.extend({
 
     // Get our initial boundaries, and then update the collection
     this._lowerBound = _.result(this, 'initialLowerBound');
+    if (typeof this.initialUpperBound  === 'function') {
+      this.initialUpperBound = this.initialUpperBound.bind(this, this._lowerBound);
+    }
```

Comment:
Overriding the fn the user provides isn't ideal. Ideally we should have something like `_.result` that lets you pass arguments. The API should be:

`execute(this, 'initialUpperBound', this._lowerBound);`

If it's a fn, it calls it, passing in `this._lowerBound`. Otherwise, it ignores the 3rd arg.